classes/AdminTab.php

```diff
@@ -1429,19 +1428,19 @@ public function displayListHeader($token = NULL)
 				<td>';
 
 		/* Display column names and arrows for ordering (ASC, DESC) */
-		if (array_key_exists($this->identifier,$this->identifiersDnd) && $this->_orderBy == 'position')
+		if (array_key_exists($this->identifier, $this->identifiersDnd) && $this->_orderBy == 'position')
 		{
 			echo '
 			<script type="text/javascript" src="../js/jquery/jquery.tablednd_0_5.js"></script>
 			<script type="text/javascript">
-				var token = \''.($token!=NULL ? $token : $this->token).'\';
+				var token = \''.($token != null ? $token : $this->token).'\';
 				var come_from = \''.$this->table.'\';
 				var alternate = \''.($this->_orderWay == 'DESC' ? '1' : '0' ).'\';
 			</script>
 			<script type="text/javascript" src="../js/admin/dnd.js"></script>
 			';
 		}
-		echo '<table'.(array_key_exists($this->identifier,$this->identifiersDnd) ? ' id="'.(((int)(Tools::getValue($this->identifiersDnd[$this->identifier], 1))) ? substr($this->identifier,3,strlen($this->identifier)) : '').'"' : '' ).' class="table'.((array_key_exists($this->identifier,$this->identifiersDnd) && ($this->_orderBy != 'position 'AND $this->_orderWay != 'DESC')) ? ' tableDnD'  : '' ).'" cellpadding="0" cellspacing="0">
+		echo '<table'.(array_key_exists($this->identifier, $this->identifiersDnd) ? ' id="'.(((int)(Tools::getValue($this->identifiersDnd[$this->identifier], 1))) ? substr($this->identifier,3,strlen($this->identifier)) : '').'"' : '' ).' class="table'.((array_key_exists($this->identifier, $this->identifiersDnd) && ($this->_orderBy != 'position '&& $this->_orderWay != 'DESC')) ? ' tableDnD'  : '' ).'" cellpadding="0" cellspacing="0">
```

Comment:
Hi, Maybe use Tools::substr and Tools::strlen, and space after $this->_orderBy != 'position '